### PR TITLE
Rmvp 170

### PR DESCRIPTION
**Pull request by de-sage**
created the post email flow

### CODE REVIEW

File: src/pages/innerPages/TeamManagement/manageTeam.vue [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
       <v-col cols="12" sm="8" md="5" lg="4" xl="3">
         <v-card class="pa-2" elevation="0">
           <span
             style="font-style: normal;
-font-weight: bold;
-font-size: 15px;
-line-height: 117.3%; color : #03DAC5;"
+                   font-weight: bold;
+                   font-size: 15px;
+                   line-height: 117.3%; color : #03DAC5;"
```
Comment (zanio) [inline src/pages/innerPages/TeamManagement/manageTeam.vue:11]: Please minimize the use of inline-styling, it makes our code had to manage

File: src/pages/innerPages/TeamManagement/manageTeam.vue [hunk 2]
```diff
@@ -26,17 +22,11 @@ To grant a priviledge, move the slider beside it to the right..
...
-                font-weight: 300;
-                font-size: 10px;
-                line-height: 117.3%; color: #000000;  margin-top : -15px"
-          >mobile devices.</p>-->
           <p
             style="font-style: normal;
                 font-weight: 300;
                 font-size: 10px;
-                line-height: 117.3%; color: #000000; "
+                line-height: 117.3%; color: #000000; margin-top : -5px"
```
Comment (zanio) [inline src/pages/innerPages/TeamManagement/manageTeam.vue:29]: Same as Above

File: src/pages/innerPages/TeamManagement/manageTeam.vue [hunk 3]
```diff
@@ -59,54 +49,182 @@ To grant a priviledge, move the slider beside it to the right..
...
+        
+      
+           <v-card-actions style="margin-top : -5px; width : 273px">
+               <p style="font-family: Work Sans;
+                  font-style: normal;
+                  font-weight: bold;
+                  font-size: 13px;
+                  line-height: 117.3%;
+                  margin-left : -8px">
+                      <!-- {{ show ? 'First Name' : 'Temimadu David'}} -->
```
Comment (zanio) [inline src/pages/innerPages/TeamManagement/manageTeam.vue:63]: Please remove unnecessary comment

File: src/pages/innerPages/TeamManagement/manageTeam.vue [hunk 4]
```diff
@@ -59,54 +49,182 @@ To grant a priviledge, move the slider beside it to the right..
...
+    </v-card-actions>
+      <p 
+      style="font-family: Work Sans;
+             font-style: normal;
+             font-weight: 300;
+             font-size: 12px;
+             line-height: 117.3%;;
+             margin-top : -23px">
+                Team Member 1 | <span>Edit Team member </span>
+    <!-- {{ show ?'Temimadu' : 'Team Member 1'   }} -->
```
Comment (zanio) [inline src/pages/innerPages/TeamManagement/manageTeam.vue:84]: Please remove unnecessary comment

**Review by zanio — CHANGES_REQUESTED** [review]

File: src/pages/innerPages/TeamManagement/manageTeam.vue (hunk 1 above)
Comment (de-sage) [reply src/pages/innerPages/TeamManagement/manageTeam.vue:11]: alright

**Comment by de-sage** [comment]
removed the comment for the inline style it is something that will take a significant amount of time and it only affects the inner pages classes so you can merge and go on with the implementation of the on boarding i have started work on the inline styles

**Pull request merged**